activemq/datadog_checks/activemq/data/conf.yaml.example

```diff
@@ -1,35 +1,95 @@
+## sample files, https://raw.githubusercontent.com/DataDog/dd-agent/5.10.1/conf.d/activemq.yaml.example
 
 instances:
+  
+  ## @param host - string - required
+  ## 
+  #
   - host: localhost
+
+  ## @param port - integer - required
+  ##
+  #
     port: 1099
-  #   user: username
-  #   password: password
-  #   process_name_regex: .*process_name.* # Instead of specifying a host, and port. The agent can connect using the attach api.
-  #                                        # This requires the JDK to be installed and the path to tools.jar to be set below.
-  #   tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar # To be set when process_name_regex is set
+
+  ## @param user - string - optional
+  ##
+  #
+  #  user: <USERNAME>
+  
+  ## @param password - string - optional
+  ## XXXXXXXXXXX
+  #
+  #  password: <PASSWORD>
+
+  ## @param process_name_regex - string - optional
+  ## XXXXXXXXXXX
+  #
+  #  process_name_regex: .*process_name.* 
+
+  ## @param tools_jar_path - string - optional
+  ## Instead of specifying a host, and port. The agent can connect using the attach api.
+  ## This requires the JDK to be installed and the path to tools.jar to be set below.
+  ## Note: It needs to be set when process_name_regex parameter is set
+  #
+  #  tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar
+  
+  ## @param name - string - optional
+  ## XXXXXXXXXXX
+  #
   #   name: activemq_instance
-  #   # java_bin_path: /path/to/java # Optional, should be set if the agent cannot find your java executable
-  #   # java_options: "-Xmx200m -Xms50m" # Optional, Java JVM options
-  #   # trust_store_path: /path/to/trustStore.jks # Optional, should be set if ssl is enabled
-  #   # trust_store_password: password
-  #   tags:
-  #     env: stage
-  #     newTag: test
+
+    ## @param java_bin_path - string - optional
+    ## java_bin_path should be set if the agent cannot find your java executable
+    #
+    #  java_bin_path: <JAVA_PATH>
+
+    ## @param java_options - string - optional
+    ## List of Java JVM options.
+    #
+    #  java_options: "-Xmx200m -Xms50m"
+
+    ## @param trust_store_path - string - optional
+    ## trust_store_path should be set if ssl is enabled.
+    ## path to your trusted store
+    #
+    #  trust_store_path: <TRUSTSTORE.JKS_PATH>
+
+    ## @param trust_store_password - string - optional
+    ## trust_store_password should be set if ssl is enabled
+    ## password for your TrustStore.jks file
+    #
+    #  trust_store_password: password
+  
+  ## @param tags  - list of key:value element - optional 
+  ## List of tags to attach to every metric, event and service check emitted by this integration.
+  ## 
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  #
+  #  tags:
+  #    - <KEY_1>:<VALUE_1>
+  #    - <KEY_2>:<VALUE_2>
 
 init_config:
+
+  ## @param is_jmx - boolean - required - default: true
```

Comment:
Not sure this and `collect_default_metrics` are required if there is a default value